doc/getting-started.md

```diff
@@ -608,40 +609,49 @@ git push origin master
 Use your crowd login when asked for credentials.
 we do this as the rundeck user, so we can accept the ssh host key.
 
-### Setup and Configure Nexus3
-#### Prepare in Minishift
-The OpenDevStack Quickstarters assume, that you will have a Nexus3 Repository Manager in place to store the artifacts, generated during build. So we have to setup a Nexus3 in the bas project.
-```
-oc new-app sonatype/nexus3 -n cd
-```
+### Prepare environment settings
+Copy the entire directory structure from `ods-configuration-sample` into `ods-configuration`and remove the .sample postfixes.
 
-After creation change to the webconsole.
-Access the base project "OpenDevStack Templates" and open the **Routes** section in the **Applications** tab in the menu.
-Click **Create Route**
-As name enter `nexus`.
-You don't need to provide a hostname. Ensure, that the route points to the `nexus3` service with the correct port.
-For the test environment we don't need secured routes, so no change is needed here.
-Click **Create** and the route is created. You should now be able to access Nexus 3 via http://nexus-cd.192.168.99.100.nip.io/
+### Setup and Configure Nexus3
 
-#### Add persistent claim
-To have a stable state for the Repository Manager between several Minishift starts, you will have to add a persistent claim to the deployment.
-First open the **Storage** menu and click **Create Storage**.
-Define a name, e.g. `nexus3-pv` and define the size. 5GiB should last for testing purposes. Finish by clicking **Create**
-Go to **Deployments** in the **Applications** menu, click the *nexus3* Deployment and choose the **Edit Yaml** action in the upper right corner in the **Actions** drop down.
-Now replace the part
-```
-      volumes:
-        - emptyDir: {}
-          name: nexus3-volume-1
-```
-with
-```
-      volumes:
-        - name: nexus3-volume-1
-          persistentVolumeClaim:
-            claimName: nexus3-pv
-```
-A new deployment will start with a mounted persistent volume.
+Amend `ods-configuration/ods-core/nexus/ocp-config/route.env` and change the domain to match your openshift/minishift domain (`nexus-cd.192.168.99.100.nip.io`)
+
+Go to `ods-core/nexus/ocp-config` - and type 
+``` bash
+tailor update
+```
+You should see a proposed list of new objects that are created - and confirm with `y`
+
+```
+Comparing templates in C:\code_bix\opendevstack_at_BIX\ods-core\nexus\ocp-config with OCP namespace cd.
```

Comment:
Maybe the path should be modified here to be more generic

Reply:
@clemensutschig Thought of Maxim, but if you don't see an issue with this, it's ok.